lib/hobbes/read/pgen/hexpr.y

```diff
@@ -229,6 +230,22 @@ MonoTypePtr makeRecType(const Record::Members& tms) {
   return Record::make(tms);
 }
 
+MonoTypePtr makePVarType(const Variant::Members &vms,
```

Comment:
Let's change `Variant::make(vms)` to use `makePVarType(...)` [in the C++ constructor for penum](https://github.com/morganstanley/hobbes/blob/90c9865cfbfb3c4c08437dcd27bd9a090028ff99/include/hobbes/lang/tylift.H#L413) too

Reply:
Should this check be added to `Variant::make` itself, just wondering?